build_elexis/assets/build_inside_container

```diff
@@ -7,19 +7,27 @@ build_root=/home/elexis
 cd $build_root
 if [ -d "$build_root/elexis-3-core" ]
 then
-  echo "Don't clone elexis-3-core"
+  echo "pull elexis-3-core"
+  cd elexis-3-core
+  git pull
+  cd ..
 else
+  echo clone elexis-3-core
   git clone https://github.com/elexis/elexis-3-core
 fi
 cd elexis-3-core
 # Add -Dmaven.test.skip=true if you have problems running the tests, eg. no X display
-echo mvn clean install  $MAVEN_OPTS_ELEXIS_3_CORE
-mvn clean install  $MAVEN_OPTS_ELEXIS_3_CORE
+echo mvn clean install  $MAVEN_OPTS_ELEXIS_3_CORE -Dmaven.test.skip=true
+mvn clean install  $MAVEN_OPTS_ELEXIS_3_CORE -Dmaven.test.skip=true
 cd $build_root
```

Comment:
Well the reason for -Dmaven.test.skip was, that the tests break the Dockerfiles: They simply don't work (At least on my 2 tested hosts, Mac OS 10.10 and Ubuntu 14.04 headless)
Without tests, the Dockerfile works for elexis-3-core (still not for elexis-3-base).

Sure, I understand your point. Docker containers as build system for the releaes absolutely must run tests. So I'd recommend, you leave my changes here out, and instead use my other subproject (elexis3factory) as a build system for home use.

The problem is: If you insist on using a build system that only works in a very limited number of environments, it is difficult for other people to join in elexis development. So, there should be an option for people who just want to use their mac or their windows or their whatever without the hassle of a difficult setup.